MobileCenter/MobileCenter/MSMobileCenter.h

```diff
@@ -96,18 +96,18 @@
  */
 + (void)setWrapperSdk:(MSWrapperSdk *)wrapperSdk;
 
-#if TARGET_OS_OSX
```

Comment:
Yeah.. This is a bad merge. I also fixed in the branch of feature/support-macos-10-9.